api/src/main/java/org/ehcache/spi/service/ServiceCreationConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.spi.service;
+
+/**
+ * Interface regrouping configuration types to be used when creating a {@link Service}.
+ *
+ * @param <T> the service type this configuration works with
+ *
+ */
+public interface ServiceCreationConfiguration<T extends Service> {
+
+  /**
+   * Indicates which service this configuration works with.
```

Comment:
`Indicates which service type this instance configures`... it's more specific than works with now, right?